src/js/node/stream.ts

```diff
@@ -4429,6 +4449,18 @@ var require_duplex = __commonJS({
 
     function Duplex(options) {
       if (!(this instanceof Duplex)) return new Duplex(options);
+
+      this._events ??= {
```

Comment:
`_events` should be created by the buggy C++ EventEmitter implementation because it has a hidden native type

When we move off of that, we should probably put this back but until then we should let that code create the `_events` object.